exporter/exporter.go

```diff
@@ -316,6 +317,45 @@ type topEntry struct {
 	Value float64
 }
 
+func getFieldsAndTags(topic string, entry map[string]interface{}, tags map[string]string, fields map[string]string) (tagsMap map[string]string, fieldsMap map[string]interface{}) {
+	tagsMap = make(map[string]string)
+	for tag, value := range tags {
+		if len(value) > 1 && strings.HasPrefix(value, "#") {
+			key := value[1:]
+			actualValue, ok := entry[key]
+			if !ok {
+				log.Printf("field %q not found for tag in topic %q message %+v", key, topic, entry)
```

Comment:
Let's also log the tag and attempted value so it's easy to find the offending line in the config.